Added changelog for version 3.10.0 (#684)
MapsIndoors/docs: src/changelogs/android/v3/CHANGELOG.md

### Security     in case of vulnerabilities.
-->

## [3.10.0] 2021-08-19

### Added

> - The MapsIndoors SDK records anonymous usage statistics and diagnostic events per default (if enabled in the CMS) and sends the logged data to a server at MapsPeople. Logging may be disabled entirely by calling MapsIndoors.disableEventLogging(true) on the SDK.

Reviewer: ```suggestion
- The MapsIndoors SDK records anonymous usage statistics and diagnostic events per default (if enabled in the CMS) and sends the logged data to a server at MapsPeople. Logging may be disabled entirely by calling `MapsIndoors.disableEventLogging(true)` on the SDK.
```

### Deprecated

- Deprecated `MapsIndoors.setOnMapsIndoorsReadyListener(onReadyListener)`. Use `MapsIndoors.addOnMapsIndoorsReadyListener(onReadyListener)` instead.

### Fixed

> - Polygon DisplayRules can now be applied to all locations with a polygon, instead of just areas

Reviewer: ```suggestion
- Polygon DisplayRules can now be applied to all Locations with a polygon, not just Areas
```

> - Livedata badges now correctly scale according to the devices pixel density

Reviewer: ```suggestion
- Live Data badges now correctly scale according to the device's pixel density
```
